package.json

```diff
@@ -47,7 +47,7 @@
     "preact": "10.19.6",
     "rollup": "^4.19.1",
     "sass": "^1.76.0",
-    "tailwindcss": "^3.3.3",
+    "tailwindcss": "^3.4.10",
```

Comment:
Tailwind 3.4 added size utilities to `min-w-*` classes, that I wanted to use here: https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.4.0